dev-tools/validation/test_check_authz_route_matrix.py

```
@@ -482,6 +3036,267 @@ def test_join_route_errors_are_emitted_in_sorted_order(self):
+    def test_unknown_route_status_is_rejected(self):
+        document = self.validator.yaml.safe_load(MATRIX.read_text(encoding="utf-8"))
+        route = route_for(document, "account-service", "BillingArtifactsTenant")
+        route["route_status"] = "target_only"
+        errors = validate_document(self.validator, document)
+        self.assertTrue(any("route_status must be one of" in error for error in errors))
+
+    def test_private_receiver_requires_exact_token_and_method_predicates(self):
+        document = self.validator.yaml.safe_load(MATRIX.read_text(encoding="utf-8"))
+        route = route_for(document, "account-service", "RefreshGameplayServiceToken")
+        policy = route["caller_policies"][0]
+        policy["token_audience"] = "internal"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token_audience"

(S105)